src/platform/plugins/shared/share/public/components/share_tabs.tsx

```diff
@@ -34,14 +36,65 @@ export const ShareMenuTabs = () => {
     tabs.push(linkTab);
   }
 
-  // Embed is disabled in the serverless offering, hence the need to check that we received it
+  // Embed is disabled in the serverless offering, hence the need to check if the embed tab should be shown
   if (
     shareMenuItems.some(({ shareType }) => shareType === 'embed') &&
     !objectTypeMeta?.config?.embed?.disabled
   ) {
     tabs.push(embedTab);
   }
 
+  /**
+   * If there is only one tab and the link is configured to allow copying the link without showing the modal,
+   * we will copy the link to the clipboard and show a success toast on copying the share link for object.
```

Comment:
It looks like this improvement is only related to the share button, and not the export one that still present an empty flyout (so probably the PR title is wrong and we are only talking about the share button)
<img width="890" alt="Screenshot 2025-06-18 at 10 00 13" src="https://github.com/user-attachments/assets/b546a784-9545-4171-88b1-bf822b01d255" />

Reply:
Thanks for the pointer... I resolved this in [`128c5e4` (#222242)](https://github.com/elastic/kibana/pull/222242/commits/128c5e41264ee89b6d311cde2b00cff96ecfb99d)